safe-global/safe-wallet-web
# WIP: Tx flow success screen

## What it solves

Resolves #2023 

## ToDos

- [x] Stop the animation on Success/Error

## How this PR fixes it
Implements the Success screen that is displayed after executing a transaction with your connected wallet.

## How to test it
1. Submit a transaction execution
2. Sign/reject with your wallet
3. Observe the Success screen progressing through the stepper

## Screenshots
<img width="1076" alt="Screenshot 2023-06-02 at 15 11 15" src="https://github.com/safe-global/safe-wallet-web/assets/32431609/8f01538d-48c9-4de1-b4b4-32e7510e522c">
<img width="1011" alt="Screenshot 2023-06-02 at 15 11 42" src="https://github.com/safe-global/safe-wallet-web/assets/32431609/6699fa54-a952-48c4-b33e-0079c9064b62">

## Checklist
